src/pages/Dashboard/index.tsx

```diff
@@ -74,7 +75,9 @@ const NewsFeedSettings: Settings = {
   speed: 500,
   slidesToShow: 1,
   slidesToScroll: 1,
-  arrows: false
+  arrows: false,
```

Comment:
@prodesert22 another thing we need to think about after the birthday is to include a back arrow in the News widget

![image](https://user-images.githubusercontent.com/3041005/151907846-f7d5427b-2ef2-4614-9b5e-e6053b329e9d.png)

When I read through all the news I sometimes want to go back. And then also perhaps a clear button to hide the news you've already read